src/main/java/core/basesyntax/exception/PasswordValidationException.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.exception;
+
+public class PasswordValidationException extends Exception {
+    private String message;
+
+    public PasswordValidationException(String message) {
+        super(message);
+        this.message = message;
```

Comment:
Do you need to store message in the exception object? 
```suggestion
    public PasswordValidationException(String message) {
        super(message);
```